lib/service/connect.go

```diff
@@ -807,7 +807,7 @@ func (process *TeleportProcess) newClient(authServers []utils.NetAddr, identity
 	// connect through a tunnel.
 	process.log.Debugf("Attempting to connect to Auth Server directly.")
 	if _, err = directClient.GetLocalClusterName(); err != nil {
-		process.log.WithError(err).Warn("Failed to connect to Auth Server directly.")
+		process.log.Debug("Failed to connect to Auth Server directly.")
```

Comment:
same as other PR, store the error and return an aggregate error message below so it's not lost